docker.go

```diff
@@ -181,6 +183,12 @@ func (p Plugin) Exec() error {
 		cmds = append(cmds, commandPull(img))
 	}
 
+	// cache export feature is currently not supported for docker driver
+	// hence we have to create docker-container driver
+	if len(p.Build.ExportCache) > 0 {
```

Comment:
if len(p.Build.ExportCache) > 0 || len(p.Build.ImportCache) > 0 {}

Reply:
Docs do not seem to indicate that docker driver is not supported. It only mentions about the restriction during exports.

https://docs.docker.com/build/cache/backends/